build.gradle

```diff
@@ -161,7 +161,7 @@ subprojects {
         findBugsAnnotationsVersion = '3.0.1'
         findBugsJsr305Version = '3.0.2'
         errorProneVersion = '2.3.2'
-        grpcVersion = '1.25.0'
+        grpcVersion = '1.27.2'
```

Comment:
upgrading to grpcVersion will break grpc metrics collection using opencensus unless grpc-census package is included. See this [release-note](https://github.com/grpc/grpc-java/releases/tag/v1.27.0).

Please update the change.log for the dependency change and link to grpc release-note in the change.log.

Reply:
done

Reply:
done